setup.py

```diff
@@ -40,5 +83,12 @@ def get_version():
     ],
     zip_safe=False,
     tests_require=["Django>=1.4.2"],
-    test_suite='runtests.runtests'
+    test_suite='runtests.runtests',
+    setup_requires=['Django>=1.4.2'],
+    include_package_data=True,
```

Comment:
I usually prefer to leave this `False`, since its functionality depends on VCS plugins. The `package_data` kwarg still works without it.